src/main/java/seedu/address/flashcard/MCQ.java

```diff
@@ -0,0 +1,27 @@
+package seedu.address.flashcard;
+
+/**
+ * Represents a mcq question.
+ * Options for the mcq will be stored in an array while the
+ * question itself will be stored as a {@code String}.
+ */
+public class MCQ implements Question{
```

Comment:
Just wondering if it would be better if `MCQ` was named `MultipleChoiceQuestion` to standardize with our `OpenEndedQuestion` class. Would be more descriptive for whoever is looking at the code base as well.

Reply:
good suggestion. will change it